providers/zendesksupport/modules.go

```diff
@@ -0,0 +1,19 @@
+package zendesksupport
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/providers/zendesksupport/metadata"
+)
+
+const (
+	// ModuleTicketing is used for proxying requests through.
+	// https://developer.zendesk.com/api-reference/ticketing/introduction/
+	ModuleTicketing common.ModuleID = "ticketing"
+	// ModuleHelpCenter is Zendesk Help Center.
+	// https://developer.zendesk.com/api-reference/help_center/help-center-api/introduction/
+	ModuleHelpCenter common.ModuleID = "help-center"
```

Comment:
@RajatPawar 

Empty module is removed. It doesn't make sense to have such.
The old comment which was copied from old code mentions proxying to be the reason for such module, it is misleading.

For example, module selection has no impact on something like this:
```
// From: examples/authconnectors/salesforce/salesforce.go
conn := createAuthConnector(ctx)
response, err := conn.Client.Get(ctx, "/services/data/v61.0/limits/")
```
Read/Write make sense only with resepect to `ticketing` and `help-center`.